tests/resources/test-types.json

```diff
@@ -11173,11 +11173,29 @@
                 "linkedIds": null,
                 "name": "VTG10 Notifiable Alteration with MR only",
                 "testTypeName": "VTG10 Notifiable Alteration with MR only",
-                "forVehicleType": "hgv",
+                "forVehicleType": ["hgv", "trl"],
                 "forVehicleSize": null,
                 "forVehicleConfiguration": null,
                 "forVehicleAxles": null,
-                "forEuVehicleCategory": null,
+                "forEuVehicleCategory": [
```

Comment:
I'll admit that I do not know how this service works, but could this effect the current **hgv** behaviour? I can see forEuVehicleCategory is populated again further down specifically for **trl**.

Reply:
That is a good point, I believe only o2, o3 and o4 are applicable to trailer. I don't _think_ this matters though, and it will do an AND filter on the vehicle type and eu vehicle category. The issue would be if we sent a trailer with the wrong eu vehicle category (such as m1). Instead of erroring out, it will assign the code, which is somewhat undesired behaviour.

Reply:
This applies the same approach we took with [CB2-7765](https://dvsa.atlassian.net/browse/CB2-7765). If that is the case, we should proliferate the change to all modified test codes as part of that ticket.